pypdf/_writer.py

```diff
@@ -3133,7 +3133,7 @@ def add_filtered_articles(
                     thr = thr.get_object()
                 if thr.indirect_reference.idnum not in self._id_translated[
                     id(reader)
-                ] and fltr.search(thr["/I"]["/Title"]):
```

Comment:
AttributeError: 'IndirectObject' object has no attribute 'get'

Reply:
I've seen... Fix for tonight... Too late yesterday...2 good PRs is not too bad for a coming back🤣🤣🤣